apps/event-manager-api/src/app/dto/create-event.dto.ts

```diff
@@ -21,4 +21,6 @@ export class CreateEventDTO {
   ticketQuantity: number;
   @IsString()
   acceptedMint: string;
+  @IsString()
+  fId: string;
```

Comment:
It seems like `fId` stands for Firebase ID. What's the difference between fId and id? why not use the ID from firebase as the "id" of the event?